packages/gluestick-cli/index.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const commander = require('commander');
+const mkdir = require('mkdirp');
+const spawn = require('cross-spawn');
+
+commander.version(
+  require(path.join(__dirname, 'package.json')).version
+);
+
+commander
+  .command('new')
+  .description('generate a new application')
+  .arguments('<appName>')
+  .option('-d, --dev <path>', 'development')
+  .action((appName, options) => {
+    mkdir.sync(path.join(process.cwd(), appName));
+    spawn.sync(
+      'npm',
```

Comment:
If it isn't too much trouble I think it would be worth creating a new issue for using `yarn` over `npm install` for the speed advantage.